components/application-operator/README.md

```diff
@@ -29,10 +29,7 @@ The Application Operator has the following parameters:
  - **appName** is the name used in controller registration. The default value is `application-operator`.
  - **domainName** is the domain name of the cluster. The default domain name is `kyma.local`.
  - **namespace** is the Namespace where the AO deploys the charts of the Application. The default Namespace is `kyma-integration`.
- - **tillerUrl** is the Tiller release server URL. The default value is `tiller-deploy.kube-system.svc.cluster.local:44134`.
- - **helmTLSKeyFile** is the path to the TLS key used for secure communication with Tiller. The default value is `/etc/certs/tls.key`.
- - **helmTLSCertificateFile** is the path to the TLS certificate used for secure communication with Tiller. The default value is `/etc/certs/tls.crt`.
- - **tillerTLSSkipVerify** disables TLS verification in communication with Tiller. The default value is `true`.
```

Comment:
I think we are missing `healthPort` flag in the README, we can add it.

Reply:
I have added description of this parameter